src/App.tsx

```diff
@@ -1,24 +1,203 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Etodos, ResponseError } from './types/enum';
+import { UserData } from './types/userData';
+import { Todo } from './types/Todo';
+import { deleteTodoOnServer, getTodos, updateTodoProp } from './api';
+import { TodoItem } from './components/Todo';
+import { TodoHeader } from './components/TodoHeader';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const defaultUser = {
+  createdAt: '2023-07-10T13:09:53.578Z',
+  email: 'gookidoo@gmail.com',
+  id: 11038,
+  name: 'Віктор Булденко',
+  phone: null,
+  updatedAt: '2023-07-10T13:09:53.578Z',
+  username: null,
+  website: null,
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [user] = useState<UserData>(defaultUser);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isUncomplete, setIsUncomplete] = useState<number>(0);
```

Comment:
this value is based on todos. No need to store it in a state - use a simple variable